docs/detections/prebuilt-rules/rule-details/aws-rds-security-group-creation.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[aws-rds-security-group-creation]]
+=== AWS RDS Security Group Creation
+
+Identifies the creation of an Amazon Relational Database Service (RDS) Security Group.
+
+*Rule type*: query
+
+*Rule indices*:
+
+* filebeat-*
+* logs-aws*
+
+*Severity*: low
+
+*Risk score*: 21
+
+*Runs every*: 10 minutes
+
+*Searches indices from*: now-60m ({ref}/common-options.html#date-math[Date Math format], see also <<rule-schedule, `Additional look-back time`>>)
+
+*Maximum alerts per execution*: 100
+
+*References*:
+
+* https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBSecurityGroup.html
+
+*Tags*:
+
+* Elastic
+* Cloud
+* AWS
+* Continuous Monitoring
+* SecOps
+* Monitoring
+
+*Version*: 1
+
+*Added ({stack} release)*: 7.14.0
+
+*Rule authors*: Elastic, Austin Songer
+
+*Rule license*: Elastic License v2
+
+==== Potential false positives
+
+A RDS security group may be created by a system or network administrator. Verify whether the user identity, user agent, and/or hostname should be making changes in your environment. Security group creations from unfamiliar users or hosts should be investigated. If known behavior is causing false positives, it can be exempted from the rule.
```

Comment:
```suggestion
An RDS security group may be created by a system or network administrator. Verify whether the user identity, user agent, and/or hostname should be making changes in your environment. Security group creations from unfamiliar users or hosts should be investigated. If known behavior is causing false positives, it can be exempted from the rule.
```